### PR TITLE
SEO-173273 - WinUI Ribbon weblink controls

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @MallikaRK ,
Worked on image alt text and renamed images.

Regards,
Sharly A.

### CODE REVIEW

File: winui/Ribbon/Getting-Started.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ this.rootGrid.Children.Add(sfRibbon);
 {% endhighlight %}
 {% endtabs %}
 
-![Add ribbon tab in ribbon](Getting-Started-images/add-ribbon-tab.png)
+![WinUI Ribbon tab added in ribbon](getting-started-images/winui-ribbon-added-tab.png)
```
Comment (MallikaRK) [inline winui/Ribbon/Getting-Started.md:98]: Change alt text like as 
Add Ribbon Tab in WinUI application.
Change image name as per alt text.

File: winui/Ribbon/Overview.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ The Syncfusion [WinUI Ribbon](https://www.syncfusion.com/winui-controls/ribbon)
 * **RibbonGallery:** A simple and flexible ribbon gallery feature displays a collection of related commands similar to the gallery feature used in Microsoft Office.
 
 
- ![Overview of WinUI SfRibbon control](Overview-images/syncfusion-winui-ribbon.png)
+ ![Overview of WinUI SfRibbon control](0verview-images/winui-ribbon-overview.png)
```
Comment (MallikaRK) [inline winui/Ribbon/Overview.md:24]: mistakenly added zero here in Image Folder Name. Check and correct it. Avoid careless mistakes in future.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: winui/Ribbon/Getting-Started.md (hunk 1 above)
Comment (Sharly-Atieno) [reply winui/Ribbon/Getting-Started.md]: Changed

File: winui/Ribbon/Overview.md (hunk 2 above)
Comment (Sharly-Atieno) [reply winui/Ribbon/Overview.md]: Removed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**